CHANGELOG.asciidoc

```diff
@@ -42,6 +42,7 @@ endif::[]
 ===== Bug fixes
 
 * Fix an async issue with long elasticsearch queries {pull}1725[#1725]
+* Fix a performance issue with AWS Lambda Support {pull}1727[#1727]
```

Comment:
I'm not sure user's will notice, but should the change deprecating[^1] use of kwargs to `capture_serverless` be called out here.

[^1]: Or perhaps not "deprecated" yet? The `PendingDeprecationWarning` below says it'll be deprecated later... as opposed to deprecated now and *remove* in a subsequent major.

Reply:
We use `DeprecationWarning` when support is removed, and `PendingDeprecationWarning` in the period leading up to its removal.